### PR TITLE
Transfer agreement form

### PR DESCRIPTION
**Pull request by TomJKing**

### CODE REVIEW

File: app/controllers/TransferAgreementController.scala [hunk 1]
```diff
@@ -3,12 +3,46 @@ package controllers
...
+      "publicRecord" -> boolean
+        .verifying("Must answer yes", b => b),
+      "crownCopyright" -> boolean
+        .verifying("Must answer yes", b => b),
+      "english" -> boolean
+        .verifying("Must answer yes", b => b),
+      "digital" -> boolean
+        .verifying("Must answer yes", b => b),
+      "droAppraisalselection" -> boolean
+        .verifying("Must answer yes", b => b),
```
Comment (MancunianSam) [inline app/controllers/TransferAgreementController.scala]: You need another capital in `droAppraisalselection` There's a few places in the file where this is so I won't comment on them all

File: test/viewsapi/FormFunctionSpec.scala [hunk 2]
```diff
@@ -60,21 +60,37 @@ class FormFunctionSpec extends FrontEndTestHelper {
...
+    }
+
+    "not render disabled when no _disabledOption" in {
+      val args: Map[Symbol, Any] = Map(Symbol("_madeUpOption") -> "true")
+
+      val result = FormFunctions.inputRenderOptions(args).disabledInput("true")
+      result shouldBe ()
+    }
+
   "setErrorClass function" should {
```
Comment (MancunianSam) [inline test/viewsapi/FormFunctionSpec.scala]: There's a new `selectedInput` method in the form functions but there's no test for these in here.

File: app/controllers/TransferAgreementController.scala (hunk 1 above)
Comment (TomJKing) [reply app/controllers/TransferAgreementController.scala]: fixed

File: test/viewsapi/FormFunctionSpec.scala (hunk 2 above)
Comment (TomJKing) [reply test/viewsapi/FormFunctionSpec.scala]: fixed

**Pull request merged**